app/views/filter/_filter.html.erb

```diff
@@ -96,6 +96,15 @@
           </div>
         </div>
 
+        <div class="filter__group mt-1/4 mb-1/1">
+          <p class="filter__label">Players</p>
+          <%= content_tag :div, nil, class: "num-player-slider", data: {
+            type: "filter",
+            min_players: (to_range(CGI.escape(params[:players])).first if params[:players].present?),
+            max_players: (to_range(CGI.escape(params[:players])).last if params[:players].present?) } %>
+          <div style="display: none" data-filter-type="players" <%= "data-value=#{CGI.escape(params[:players])}" if params[:players].present? %>></div>
```

Comment:
This could also be a rails tag rather than a html tag. Also, there's a `hidden` class you could use instead of `style="..."`, not strictly necessary, but a little nicer in my opinion.